packages/expo-document-picker/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "expo-document-picker",
+  "version": "1.0.0",
+  "description": "Expo Document Picker standalone module",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "lint": "expo-module lint",
+    "test": "expo-module test",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "expo-module": "expo-module"
+  },
+  "keywords": [
+    "react-native",
+    "expo",
+    "expo-document-picker",
+    "document-picker"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git"
+  },
+  "homepage": "https://github.com/expo/expo/tree/master/packages/expo-document-picker",
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "dependencies": {
+    "uuid": "~3.3.2"
+  },
+  "peerDependencies": {
+    "expo-core": "~2.0.0"
+  },
```

Comment:
If this package doesn't depend on `expo-module-scripts`, I don't think `scripts` will run ok